src/documents/tutorial-userInterfaceOptions/WorkingWithUserInterfaceOptions.md

```diff
@@ -0,0 +1,182 @@
+    * white on black
+    * black on yellow
+    * yellow on black
+* low contrast
+    * light grey on dark grey
+
+The themes attempt to remove any other colours from the interface, along with gradients, shadows, background images, and other styling that may reduce legibility. This type of contrast interface can be extremely helpful for users with different forms vision needs.
+
+UI Options applies contrast themes by adding a special class to the `<body>` element of the document and providing styles that are scoped to that class. You will likely need to create a few more special classes, scoped to the theme, for some areas of your site.
+
+To help you in creating a website that works well with UI Options, here are a few basic guidelines:
+
+* [Make cosmetic graphics background images in CSS; create contrast versions; and use CSS to swap in the contrast versions when the themes are activated.](./UsingImagesInContrastModes.md)
+* [Improve clarity and legibility by removing gradients, shadows, and other styling effects.](./StyleEffectsAndLegibilityInContrastModes.md)
+* [Double-check use of CSS "content" property](./ContentPropertyHighContrast.md)
+
+## How to define special styles for contrast themes
+
+If something doesn't transform into contrast mode automatically when using the contrast adjustment, here's how you define styles to address those cases:
+
+### Scope your styles to the theme name
+
+UI Options will add the theme name as a class to the `<body>` element.
+
+* yellow (`#FFFF00`) on black (`#000000`): `<body class="fl-theme-yb">`
+* white (`#FFFFFF`) on black (`#000000`): `<body class="fl-theme-wb">`
+* black (`#000000`) on yellow (`#FFFF00`): `<body class="fl-theme-by">`
+* black (`#000000`) on white (`#FFFFFF`): `<body class="fl-theme-bw">`
+* light grey (`#BDBDBB`) on dark grey (`#555555`): `<body class="fl-theme-lgdg">`
+
+Scope your styles using the class name as shown in the following example. This example is for the yellow on black theme, which uses the class name `fl-theme-yb`:
+
+```html
+<body class="fl-theme-yb">
+    <div class="toolbar">
+        <a class="helpLink">Help</a>
+    </div>
+</body>
+```
+
+```css
+/* default style for button */
+.toolbar .helpLink {
+    outline: solid 0.2rem #008cba;
+    color: #008cba;
+    background: #efefef;
+    text-shadow: 1px 1px 0 #FFFFFF;
+}
+
+/* high contrast for button */
+.fl-theme-yb .toolbar .helpLink {
+    outline: solid 0.2rem #ff0;
+    color: #ff0;
+    background: #000;
+    text-shadow: none;
+}
+```
+<div class="infusion-docs-note"><strong>Note:</strong>
+    <p>
+        In some cases you may have to use the <code>!important</code> rule with your contrast styles to make them work. For example: <code>color: #ff0 !important;</code>.
+    </p>
+
+    <p>
+        Using <code>!important</code> should be used carefully as documented in <a href="https://css-tricks.com/when-using-important-is-the-right-choice/">this article on CSS-tricks.com</a>.
+    </p>
+</div>
```

Comment:
The second `<p>` is showing up as a code block in Github. Try removing the empty line between the two `<p>` blocks.